src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4680,4 +4680,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_PeWritingFailure" xml:space="preserve">
     <value>An error occurred while writing the output file: {0}</value>
   </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning" xml:space="preserve">
+    <value>Attribute '{0}' is ignored when public signing is specified.</value>
+  </data>
+  <data name="WRN_AttributeIgnoredWhenPublicSigning_Title" xml:space="preserve">
```

Comment:
Yes, still fails when the code is at the bottom. As per the comment in that test, it's checking to see if the Title exists, which requires a separate localizable string because it's displayed to the end user without any arguments as part of the `.ruleset` interactive editor VS thingy. This is an old article, but shows a screenshot of it (the "Name" column in the second to last image) - https://blogs.msdn.microsoft.com/habibh/2009/08/12/create-your-own-code-analysis-rule-set-using-visual-studio-team-system-2010/ (still behaves the same in 2015, I just opened up `./src/Compilers/CSharp/CSharpCodeAnalysisRules.ruleset` with it). The Title may be used elsewhere, that's just one of them.